plugins/nf-co2footprint/src/resources/CO2FootprintReportTemplate.html

```diff
@@ -32,11 +32,40 @@
+  }
+  .jumbotron .container {
+    background-color: rgba(255,255,255,0.8);
+    border-radius: 0.3rem;
+    padding: 1rem 2rem;
+  }
+  .metric-card svg {
+    margin: 0.5rem auto;
+    height:50px;
+  }
+  .metric {
+    font-size: 2.5rem;
```

Comment:
Rough guess (without testing) to reduce the font size so that it doesn't wrap onto multiple lines.
```suggestion
    font-size: 2rem;
```